strider/fetcher.py

```diff
@@ -128,6 +128,7 @@ async def generate_from_kp(
             )
             onehop_response = await kp.solve_onehop(
                 onehop_qgraph,
+                last_hop=len(qgraph["edges"]) == 1,
```

Comment:
I understand why we want to keep track of last_hop, but I dont understand this logic in the check. There is only one qedge?

Reply:
As we work through the query one hop at a time, we're popping off each qedge as we go. This checks that we're on to the last hop.